test/inductor/test_standalone_compile.py

```diff
@@ -96,6 +97,18 @@ def test_inductor_via_export2(self):
         actual = mod_opt(inp)
         self.assertEqual(actual, correct)
 
+    def test_inductor_via_op_with_multiple_outputs(self):
+        op_name = "torch.ops.aten.native_layer_norm.default"
+        op = eval(op_name)
+        x1 = torch.randn((2, 512, 128))
+        x2 = [128]
+        x3 = torch.randn((128))
+        x4 = torch.randn((128,))
+        x5 = 1e-6
+        mod, inp = gen_gm_and_inputs(op, (x1, x2, x3, x4, x5), {})
```

Comment:
can we pass `torch.ops.aten.native_layer_norm.default` directly to `gen_gm_and_inputs`?

Reply:
Fixed.